daphne/src/lib.rs

```diff
@@ -225,6 +236,25 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_batch_duration`, `min_batch_interval_start`,
+// and `max_batch_interval_end` should be reflected to the DAP spec.
+#[derive(Deserialize, Serialize)]
+pub struct DapGlobConfig {
+    /// Maximum window size permitted in CollectReq.
+    /// Prevents Collectors from requesting wide range or reports.
+    pub max_batch_duration: u64,
+
+    /// Lower bound of an acceptable batch window in CollectReq.
```

Comment:
Let's avoid overloading the term "batch window", which has a different definition in the spec.

```suggestion
    /// Lower bound of an acceptable batch interval for collect requests.
```

Reply:
Good point. I've also updated the comment so that it describes what these variables are in more detail.